.github/workflows/ci.yml

```diff
@@ -64,6 +64,16 @@ on:
         required: false
         type: boolean
         default: false
+      playwright-docker-compose-file:
+        required: false
+        type: string
+        default: docker-compose.yml
+        description: Path to the docker-compose file to use for testing
```

Comment:
Nit: Have you considered maybe allowing to take a comma-separated list of file names as well, instead of just one name? We may have some scenarios where we have to run more than one file like this, but I'm not sure:

```bash
docker compose -f docker-compose.yml -f docker-compose.e2e.yml up -d
```